app/models/user.rb

```diff
@@ -35,8 +35,8 @@ class User < ActiveRecord::Base
   scope :key_members, -> { where(state: 'key_member') }
   scope :voting_members, -> { where(state: 'voting_member') }
 
-  scope :all_members, -> { where(state: %w(member key_member voting_member)) }
-
+  scope :all_members, -> { where(state: %w(member key_member voting_member attendee)) }
```

Comment:
We actually don't want attendees to be considered members, because members have access to see all other members + all applications. It's possible that some day we'll want attendees to be able to see a list of all the other confirmed attendees, but that's a future feature.

"Members" should probably become something like "Organizer" or "Volunteer" in this app.